homeassistant/components/yessssms/notify.py

```diff
@@ -8,34 +8,50 @@
 
 from homeassistant.components.notify import PLATFORM_SCHEMA, BaseNotificationService
 
+from .const import CONF_PROVIDER
+
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_RECIPIENT): cv.string,
+        vol.Optional(CONF_PROVIDER, default="YESSS"): cv.string,
     }
 )
 
 
 def get_service(hass, config, discovery_info=None):
     """Get the YesssSMS notification service."""
-    return YesssSMSNotificationService(
-        config[CONF_USERNAME], config[CONF_PASSWORD], config[CONF_RECIPIENT]
-    )
+    from YesssSMS import YesssSMS
```

Comment:
Please move 3rd party imports to the top of the module.

Reply:
Has that changed? wasn't the import in the class part of 'not having everything installed'?

Reply:
This changed when we introduced manifest.json.

Reply:
thanks for the detailed review